### PR TITLE
test: new rendering scenarios

### PR DESCRIPTION
**Pull request by cbt1**

### CODE REVIEW

File: test/rendering/render.mjs-snapshots/2-rows-2-columns-Pseudo-dim-as-last-row-Styling-panel-changes-Totals-above-firefox-linux.png
Comment (cbt1) [inline test/rendering/render.mjs-snapshots/2-rows-2-columns-Pseudo-dim-as-last-row-Styling-panel-changes-Totals-above-firefox-linux.png:1]: Some of the fonts that are used in this fixture does not exist in the docker container. Need to fix this.

**Comment by haxxmaxx** [comment]
So this is 12 scenarios. I'm guessing that they come from the different cases in figma? if we introduce safari as well, that is 32 snapshots... What are your thoughts on this? I'd say it isn't an optimal amount, but might not be any case that we can remove?

**Comment by haxxmaxx** [comment]
> > So this is 12 scenarios. I'm guessing that they come from the different cases in figma? if we introduce safari as well, that is 32 snapshots... What are your thoughts on this? I'd say it isn't an optimal amount, but might not be any case that we can remove?
> 
> It the same amount as before but I think there is room to remove maybe 2-3 scenarios

Yeah I think that would be nice, is it a lot of work to rearrange the fixtures? Or do we want to get this in first?

**Comment by cbt1** [comment]
> > > So this is 12 scenarios. I'm guessing that they come from the different cases in figma? if we introduce safari as well, that is 32 snapshots... What are your thoughts on this? I'd say it isn't an optimal amount, but might not be any case that we can remove?
> > 
> > 
> > It the same amount as before but I think there is room to remove maybe 2-3 scenarios
> 
> Yeah I think that would be nice, is it a lot of work to rearrange the fixtures? Or do we want to get this in first?

No it's just a matter of deleting the fixture and snapshot files. So no problem. Will delete 2-3 that I dont think are really really needed

**Comment by cbt1** [comment]
@haxxmaxx Removed 3 of the more basic scenarions

**Review by haxxmaxx — APPROVED** [review]

File: test/rendering/README.md [hunk 1]
```diff
@@ -41,25 +41,3 @@ Sometimes tests might break, if you are certain no UI changes have been made jus
 Run `yarn start:fixture`
 
 Open in browser `http://localhost:<port>/render?fixture=./<name-of-fixture>.fix.js&theme=default`
-
```
[the remark this answers is not in the document]
Comment (cbt1) [reply test/rendering/README.md:44]: The description is now part of the fixture name and as subtitle on the chart itself. So clearly visible in the snapshots.

**Pull request merged**